file_transfer_agent.go

```diff
@@ -704,3 +767,49 @@ type snowflakeProgressPercentage struct {
 	done            bool
 	startTime       time.Time
 }
+
+func (spp *snowflakeProgressPercentage) call(bytesAmount int) {
```

Comment:
Seems we don't have tests or use the progress code? In python it is used - thus tested in snowsql but we might need tests in golang implementation for ourselves.

Reply:
I'll be adding more tests!